### ISSUE TITLE
No longer works with Ember > 1.8.0

### ISSUE THREAD

**Issue by kmmathis** [thoughtindustries/ti-ember-sortable#4]
Ember 1.8 removes the metamorph tags, which seems to break this package. Will update this issue if I learn anything more.

**Comment by adamsrog** [comment]
Experiencing the same problem.  Has anyone figured out a fix?

**Comment by btecu** [comment]
There's no need of this. Just import Sortable and call Sortable.create(ul_id_name + extra_options) in your component.

**Comment by chrisvariety** [comment]
Agreed, for 1.8 and above there are no metamorph tags, so you should be able to use any off-the-shelf Sortable library. Of course pulls welcome! :wink:
